polus-image-registration-plugin/src/image_registration.py

```diff
@@ -0,0 +1,477 @@
+import cv2
+import numpy as np
+from PIL import Image
```

Comment:
Is pillow actually used in the code?

Reply:
No, it wasn't being used. Changes have been made.